### DISCUSSION TITLE
My email can not receive the One-time Password when Login

### DISCUSSION THREAD

**Discussion by TomasHawKing** [npm/feedback#936] [Registry] [answered]
A few days ago I registered an npm account with a C...._SDK@.***.com email, but when I logged in, I found that I could not receive the One-time Password

**Comment by TomasHawKing** [comment]
Excuse me, has anyone else encountered this problem? Is it solved now? Looking forward to your response

**Comment by MylesBorins** [comment, marked as answer]
Please open a ticket with support at https://npmjs.com/support
